lib/elsa/group/manager.ex

```diff
@@ -246,8 +272,40 @@ defmodule Elsa.Group.Manager do
     end)
   end
 
+  defp start_workers(state, generation_id, assignments) do
+    Enum.reduce(assignments, state.workers, fn assignment, workers ->
```

Comment:
Why a reduce instead of an each? Am I missing an accumulation or is this just a stylistic choice?

Reply:
`WorkerManager.start_worker` finished by merging the info about the new process into the manager’s state, so the `state.workers` is the accumulator here.

Reply:
Ah. I should have dipped into that function. I see it now; thanks.